src/test/java/com/polishuchenko/bookstore/controller/BookControllerTest.java

```diff
@@ -0,0 +1,269 @@
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class BookControllerTest {
+    private static MockMvc mockMvc;
+    private static final Long VALID_ID = 1L;
+    private static final Long INVALID_ID = 99L;
+    private static Book kobzar;
+    private static BookDto kobzarDto;
+    private static CreateBookRequestDto kobzarRequestDto;
+    private static CreateBookRequestDto invalidRequestDto;
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @BeforeAll
+    static void beforeAll(@Autowired WebApplicationContext applicationContext) {
+        mockMvc = MockMvcBuilders
+                .webAppContextSetup(applicationContext)
+                .apply(springSecurity())
+                .build();
+
+        kobzar = new Book();
+        kobzar.setId(VALID_ID);
+        kobzar.setTitle("Kobzar");
+        kobzar.setAuthor("Taras Shevchenko");
+        kobzar.setIsbn("978-3-16-148410-0");
+        kobzar.setPrice(BigDecimal.valueOf(50));
+        kobzar.setDescription("some description");
+        kobzar.setCoverImage("some link");
+
+        kobzarRequestDto = new CreateBookRequestDto();
+        kobzarRequestDto.setTitle(kobzar.getTitle());
+        kobzarRequestDto.setAuthor(kobzar.getAuthor());
+        kobzarRequestDto.setIsbn(kobzar.getIsbn());
+        kobzarRequestDto.setPrice(kobzar.getPrice());
+        kobzarRequestDto.setDescription(kobzar.getDescription());
+        kobzarRequestDto.setCoverImage(kobzar.getCoverImage());
+        kobzarRequestDto.setCategoriesIds(new HashSet<>());
+
+        invalidRequestDto = new CreateBookRequestDto();
+        invalidRequestDto.setTitle("");
+        invalidRequestDto.setAuthor(kobzar.getAuthor());
+        invalidRequestDto.setIsbn(kobzar.getIsbn());
+        invalidRequestDto.setPrice(kobzar.getPrice());
+        invalidRequestDto.setDescription(kobzar.getDescription());
+        invalidRequestDto.setCoverImage(kobzar.getCoverImage());
+
+        kobzarDto = new BookDto();
+        kobzarDto.setId(kobzar.getId());
+        kobzarDto.setTitle(kobzar.getTitle());
+        kobzarDto.setAuthor(kobzar.getAuthor());
+        kobzarDto.setIsbn(kobzar.getIsbn());
+        kobzarDto.setPrice(kobzar.getPrice());
+        kobzarDto.setDescription(kobzar.getDescription());
+        kobzarDto.setCoverImage(kobzar.getCoverImage());
+    }
+
+    @WithMockUser(username = "user")
+    @Test
+    @Sql(
+            scripts = {
+                    "classpath:database/books/add-book-kobzar-to-book-table.sql"
+            }, executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD
```

Comment:
If you have only 1 script, {} usage are redundant. Also, that's better to make a script string constants.